test/.eslintrc.json

```diff
@@ -1,5 +1,8 @@
 {
     "env": {
         "mocha": true
+    },
+    "globals": {
+        "mat4": true
```

Comment:
why this is global?

Reply:
mat4 = gl-matrix. It is a dependency of CartoMap, and it is loaded from node_modules with `<script src="<%- glmatrix %>"></script>`